Tokens/tokentofriend/clients/build.gradle

```diff
@@ -34,6 +35,7 @@ dependencies {
     implementation "org.apache.logging.log4j:log4j-slf4j2-impl:${log4j_version}"
     implementation "org.apache.logging.log4j:log4j-web:${log4j_version}"
     implementation "org.slf4j:jul-to-slf4j:$slf4j_version"
+    implementation "org.bouncycastle:bcprov-jdk18on:1.75"
```

Comment:
@jakubzadroga This is the wrong version of BC, we upgraded to LTS version.

Reply:
done